cfgov/cfgov/settings/base.py

```diff
@@ -237,7 +237,11 @@
     ("ht", _("Haitian Creole")),
 )
 
-LOCALE_PATHS = (os.path.join(PROJECT_ROOT, "locale"),)
+# Add the root cfgov/locale/ and the Django project cfgov/cfgov/locale
+# directories to LOCALE_PATHS
```

Comment:
```suggestion
# Add the Django project cfgov/cfgov/locale/ directory to LOCALE_PATHS.
```

Reply:
Ah, I think that file was a merge relic from the retirement PR.